FirebasePerformance/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+#Unreleased
+- [fixed] Fix a crash related to registering for notifications when the app is inbetween foreground or background states. (#13174)
```

Comment:
nit: space between "in between" or remove "in"

Reply:
Done.